modules/nw-operator-cr.adoc

```diff
@@ -23,239 +23,300 @@ endif::[]
-    ...
-  kubeProxyConfig: <4>
-    iptablesSyncPeriod: 30s <5>
-    proxyArguments:
-      iptables-min-sync-period: <6>
-      - 0s
+  serviceNetwork:
+  - 172.30.0.0/14
 ----
-<1> A list specifying the blocks of IP addresses from which pod IP addresses are
-allocated and the subnet prefix length assigned to each individual node.
 
-<2> A block of IP addresses for services. The OpenShift SDN Container Network Interface (CNI) network provider supports only a single IP address block for the service network.
+This value is inherited from `Network.config.openshift.io` object named `cluster` during cluster installation.
+ifndef::operator[]
+The value is specified in the `install-config.yaml` file.
+endif::operator[]
+
+|`spec.defaultNetwork`
+|`object`
+|Configures the Container Network Interface (CNI) cluster network provider for the cluster network.
 
-<3> Configures the default CNI network provider for the cluster network.
+|`spec.kubeProxyConfig`
+|`object`
+|
+The parameters for this object specify the `kube-proxy` configuration.
+If you do not specify the parameter values, the Cluster Network Operator applies the displayed default parameter values.
+If you are using the OVN-Kubernetes cluster network provider, the kube-proxy configuration has no effect.
 
-<4> The parameters for this object specify the Kubernetes network proxy (kube-proxy) configuration. If you are using the OVN-Kubernetes default CNI network provider, the kube-proxy configuration has no effect.
+|====
 
-<5> The refresh period for `iptables` rules. The default value is `30s`. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package] documentation.
-+
-NOTE: Because of performance improvements introduced in {product-title} 4.3 and greater, adjusting the `iptablesSyncPeriod` parameter is no longer necessary.
+[discrete]
+[id="nw-operator-cr-defaultnetwork_{context}"]
+=== defaultNetwork object configuration
 
-<6> The minimum duration before refreshing `iptables` rules. This parameter ensures that the refresh does not happen too frequently. Valid suffixes include `s`, `m`, and `h` and are described in the link:https://golang.org/pkg/time/#ParseDuration[Go time package].
-endif::operator[]
+The values for the `defaultNetwork` object are defined in the following table:
+
+.`defaultNetwork` object
+[cols=".^3,.^2,.^5",options="header"]
+|====
+|Field|Type|Description
 
+|`type`
+|`string`
+|Either `OpenShiftSDN` or `OVNKubernetes`. The cluster network provider is selected during installation. This value cannot be changed after cluster installation.
 ifdef::openshift-origin[]
 [NOTE]
 ====
 {product-title} uses the OVN-Kubernetes Container Network Interface (CNI) cluster network provider by default.
 ====
 endif::openshift-origin[]
 
+|`openshiftSDNConfig`
+|`object`
+|This object is only valid for the OpenShift SDN cluster network provider.  This value cannot be changed after cluster installation.
```

Comment:
It would be better to clarify on indivudual sub-values that they can't be changed after installation, rather than saying it here. (In particular, MTU _will_ be changeable after installation soon, but the other values still won't be.)

Reply:
MTU can be changed in post installation will be supported in 4.9 I remembered.